src/main/java/lv/acodemy/homeworks/SecondHomeWork.java

```diff
@@ -0,0 +1,59 @@
+package lv.acodemy.homeworks;
+
+import java.util.Scanner;
+import java.util.Arrays;
+
+public class SecondHomeWork {
+    public static void main(String[] args) {
+        String [] country = {"Latvia", "Estonia", "Lithuania", "Georgia", "Russia",  "Germany", "Turkey", "Poland", "Norway", "Spain"};
+
+
+        for (int i = 0; i < country.length; i++) {
+            System.out.println(country[i]);
+        } if(!country.equals("Lavtia")) {
+            System.out.println("Latvia is my motherland");
+        }
+
+        String [] sentence  = {"Sometimes", "it", "pays", "to", "stay", "in", "bed", "on", "Monday,", "rather", "than", "spending", "the", "rest", "of", "the", "week", "debugging", "Monday’s", "code."};
+
+        System.out.println(Arrays.toString(sentence));
```

Comment:
You have to print everything that is inside sentence array using loop and in one line using print() method